tower-balance/src/choose/p2c.rs

```diff
@@ -80,7 +93,7 @@ mod tests {
                 return TestResult::discard();
             }
 
-            let (a, b) = PowerOfTwoChoices::new(rand::thread_rng()).random_pair(n);
+            let (a, b) = PowerOfTwoChoices::<ThreadRng>::default().random_pair(n);
```

Comment:
IS `<ThreadRng>` needed here?

Reply:
i was surprised that, for some reason, the compiler needed it. i think it may somehow be related to quickcheck...